Project_source/app/src/main/java/android/clicker/school_live_simulator/FriendsCourseState.kt

```diff
@@ -0,0 +1,16 @@
+package android.clicker.school_live_simulator
+
+open class FriendsCourseState : NullComputerCourseState() {
+    override val price: Int = 0
```

Comment:
@yevGen-S, please, talk with @Bazalii and set base price value in fields with same names.
It should be any values, but would be better if prices values will be organized in arithmetic or geometric progression.

Reply:
Okay, lets add values from friend's course and let them be: 100, 300, 900, 2700, 8100

Reply:
Fixed